Code sorted into logical structures. Some sorting still needs to be done.

jfsehuanes opened the pull request:
Code was sorted into several scripts, each containing a logical structure. Main folders were also renamed and a recycle_bin folder was created for code we are not totally sure if we want to get rid of. There is still sorting to be done, though. Good candidates are all plotting functions; pack them into a separate script!

jfsehuanes committed:
Code has been separated into main structures and it ran OK so far. The code now runs with the script thunderfish.py. Still need to check if all functions used in all new scripts are necessary. If not, store them into dump_functions.py

jfsehuanes committed:
renamed main folders

jfsehuanes committed:
recicle bin created. For code we might not have the 'cojones' to delete... XD

jfsehuanes committed:
Typo fixed

fabiansinz commented:
Don't  use a recycle bin folder. Just delete it. The git history has all the previous code if you still need it.

I know this is not part of this pull request but: Don't use a Makefile to install the package. Create a `setup.py` (see issue #5 ). If you use a Makefile you cut out most windows users. If you use a proper `setup.py` everybody can install your package using pip for example.

jfsehuanes committed:
figure and .npy array temporal output shifted to another folders outside the code/ folder. The plan now is to analyze each recording individually and get the analysis done faster using GNU-Parallel. Some code sorting still needs to be done for this to happen.

jgrewe committed:
rename code directory to thunderfish

jgrewe committed:
[thunderfish] move audioread to import block

jgrewe committed:
[thunderfish] move argchecks to if __name__  ==__main__

... to make it callable from other within python and not only from
command line

jgrewe committed:
[packaging] add init.py that imports only the fishtracker

jgrewe committed:
[setup.py] create setup.py

jfsehuanes committed:
MAYOR COMMIT. Code now analyses each recording independently. Thereby we can create a list of jobs for executing with GNU-parallel. The jobs are first created running the script create_parallel_jobs.py. Then the jobs are executed by running following command in the command line: cat task_list.txt | parallel -–max-procs='# of processors to be used'. The output of the code was also sorted and reorganized (this should have had an independent commit…)

jgrewe committed:
[thunderfish] organize imports

jgrewe committed:
[thunderfish] remove prog and config var that are unused, bugfix

jfsehuanes committed:
.gitignore updated not to track .txt files.

jgrewe committed:
[FishTracker] remove dependency from sys.argv, add filename field

jgrewe committed:
[FishTracker.py] some style fixes

jgrewe committed:
remove .idea file

jgrewe committed:
Merge branch 'sort_code' of https://github.com/jfsehuanes/thunderfish into sort_code

Conflicts:
	thunderfish/FishTracker.py
	thunderfish/thunderfish.py

jgrewe committed:
[Auxiliary] change default for output folder

jgrewe committed:
[thunderfish] add output path argument

jfsehuanes committed:
Merge pull request #1 from jgrewe/sort_code

Sort code
